tests/repository_data/targets_simpler.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Metadata files used by the tests are usually saved in `tests/repository_data/repository/*`.  Is there a reason why they are saved in `repository_data/*`?